examples/express/package.json

```diff
@@ -8,13 +8,12 @@
     "zipkin:client": "cross-env EXPORTER=zipkin ts-node src/client.ts",
     "jaeger:server": "cross-env EXPORTER=jaeger ts-node src/server.ts",
     "jaeger:client": "cross-env EXPORTER=jaeger ts-node src/client.ts",
-    "compile": "tsc -p .",
-    "setup": "cd ../../../../ && npm ci && cd plugins/node/opentelemetry-instrumentation-express && npm run compile && cd examples && npm run compile"
```

Comment:
reviewer note: (/cc @JamieDanielson) this "setup" script is no longer necessary. Now that the examples are not part of the workspace, the user just needs to `npm install`.